packages/iov-tendermint-rpc/src/requests.ts

```diff
@@ -161,25 +159,6 @@ export interface ValidatorsRequest {
   };
 }
 
-// DefaultParams provides pass-through for calls with no params
```

Comment:
Okay, these are boilerplate stuff... figured to be DRY.
They have no arguments every, but sure, can pull them out.

Reply:
Yeah, I see. With that DRY I had to jump back and forth all the time when trying to follow the code. I guess there is a cost of DRY: indirection.

Reply:
Fair enough.